<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PersonalInfoForm</class>
 <widget class="QWidget" name="PersonalInfoForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>443</width>
    <height>468</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>个人信息</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <spacer name="topSpacer">
     <property name="orientation">
      <enum>Qt::Orientation::Vertical</enum>
     </property>
     <property name="sizeHint">
      <size>
       <width>20</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QFrame" name="infoFrame">
     <property name="styleSheet">
      <string notr="true">
        QFrame {
            background: #ffffff;
            border-radius: 16px;
            padding: 24px;
        }
        QLabel {
            font-size: 14px;
            color: #374151;
            background: transparent;
            margin: 0px;
            padding: 0px;
        }
        QLineEdit, QSpinBox, QComboBox {
            border: 1px solid #d1d5db;
            border-radius: 8px;
            padding: 6px;
            font-size: 14px;
            background: #f9fafb;
        }
        QLineEdit:focus, QSpinBox:focus, QComboBox:focus {
            border: 1px solid #3b82f6;
            background: #f9fafb;
        }
        QPushButton {
            border: none;
            border-radius: 8px;
            padding: 10px;
            font-size: 15px;
            font-weight: bold;
        }
        QLineEdit, QCheckBox {
            color: black;
        }
        #editButton {
            background: #e5e7eb;
            color: #111827;
        }
        #editButton:hover {
            background-color: #d1d5db;
        }
        #saveButton {
            background-color: qlineargradient(x1:0, y1:0, x2:1, y2:0,
                                             stop:0 #3b82f6, stop:1 #2563eb);
            color: white;
        }
        #saveButton:hover {
            background-color: #1d4ed8;
        }
        #cancelButton {
            background: #e5e7eb;
            color: #111827;
        }
        #cancelButton:hover {
            background-color: #d1d5db;
        }
      </string>
     </property>
     <layout class="QVBoxLayout" name="frameLayout">
      <item>
       <widget class="QLabel" name="titleLabel">
        <property name="styleSheet">
         <string notr="true">font-size:22px; font-weight:bold; margin-bottom:20px;</string>
        </property>
        <property name="text"><string>个人信息</string></property>
        <property name="alignment"><set>Qt::AlignmentFlag::AlignCenter</set></property>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="accountGroupBox">
        <property name="title"><string>账号信息</string></property>
        <layout class="QFormLayout" name="accountFormLayout">
         <property name="fieldGrowthPolicy"><enum>QFormLayout::ExpandingFieldsGrow</enum></property>
         <item row="0" column="0">
          <widget class="QLabel" name="labelUsername"><property name="text"><string>用户名：</string></property></widget>
         </item>
         <item row="0" column="1">
          <widget class="QLineEdit" name="usernameEdit">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Fixed"><horstretch>0</horstretch><verstretch>0</verstretch></sizepolicy>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="labelEmail"><property name="text"><string>邮箱：</string></property></widget>
         </item>
         <item row="1" column="1">
          <widget class="QLineEdit" name="emailEdit">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Fixed"><horstretch>0</horstretch><verstretch>0</verstretch></sizepolicy>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="personalGroupBox">
        <property name="title"><string>基础信息</string></property>
        <layout class="QFormLayout" name="formLayout">
         <property name="fieldGrowthPolicy"><enum>QFormLayout::ExpandingFieldsGrow</enum></property>
         <item row="0" column="0">
          <widget class="QLabel" name="labelName"><property name="text"><string>姓名：</string></property></widget>
         </item>
         <item row="0" column="1">
          <widget class="QLineEdit" name="nameEdit">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Fixed"><horstretch>0</horstretch><verstretch>0</verstretch></sizepolicy>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="labelIdNumber"><property name="text"><string>身份证号：</string></property></widget>
         </item>
         <item row="1" column="1">
          <widget class="QLineEdit" name="idNumberEdit">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Fixed"><horstretch>0</horstretch><verstretch>0</verstretch></sizepolicy>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="labelPhoneNumber"><property name="text"><string>手机号：</string></property></widget>
         </item>
         <item row="2" column="1">
          <widget class="QLineEdit" name="phoneEdit">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Fixed"><horstretch>0</horstretch><verstretch>0</verstretch></sizepolicy>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="buttonLayout">
        <item>
         <widget class="QPushButton" name="editButton"><property name="text"><string>编辑</string></property></widget>
        </item>
        <item>
         <widget class="QPushButton" name="saveButton"><property name="visible"><bool>false</bool></property><property name="text"><string>保存</string></property></widget>
        </item>
        <item>
         <widget class="QPushButton" name="cancelButton"><property name="visible"><bool>false</bool></property><property name="text"><string>取消</string></property></widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="bottomSpacer">
     <property name="orientation"><enum>Qt::Orientation::Vertical</enum></property>
     <property name="sizeHint"><size><width>20</width><height>40</height></size></property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
